plugins/docker/docker_memory

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/perl -w
+# -*- perl -*-
+
+=head1 NAME
+
+docker_memory - Munin plugin to monitor docker container memory usage.
+
+=head1 APPLICABLE SYSTEMS
+
+Should work on any Linux system that has docker support.
+
+=head1 CONFIGURATION
+
+Root privilege required to execute docker command.
+
+1. Create a new file named "docker" inside the folder /etc/munin/plugin-conf.d/
+2. Docker file content:
+
+[docker_memory]
+user root
+
+=head1 MAGIC MARKERS
+
+  #%# family=auto
+  #%# capabilities=autoconf
```

Comment:
This marker tells munin-plugin-configure that the plugin can be run with "autoconf" command line argument.

Please see http://munin-monitoring.org/wiki/PluginConcise#autoconf
.